core/adapters/multiply.go

```diff
@@ -3,36 +3,33 @@
 package adapters
 
 import (
+	"encoding/json"
 	"fmt"
 	"math/big"
-	"strconv"
 
 	"chainlink/core/store"
 	"chainlink/core/store/models"
-	"chainlink/core/utils"
 )
 
-// Multiplier represents the number to multiply by in Multiply adapter.
-type Multiplier float64
+// Multiply holds the a number to multiply the given value by.
+type Multiply struct {
+	Times *big.Float `json:"-"`
```

Comment:
What does this '-' do?

Reply:
It says field is ignored for marshalling JSON.  I added it to signal JSON handling is done elsewhere, but its not required.